prow/config.yaml

```diff
@@ -5,7 +5,7 @@ plank:
     '*': https://prow.istio.io/view/gcs/
     istio-private: https://prow-private.istio.io/view/gcs/
   pod_pending_timeout: 15m
-  pod_unscheduled_timeout: 5m
+  pod_unscheduled_timeout: 30m
```

Comment:
This seems a little excessive to me since it will make unschedulable jobs sit for 30mins before failing and autoscaling probably shouldn't take that long. This is easy to tweak though.